src/main/resources/assets/botania/models/block/shapes/tiny_potato.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parent": "minecraft:block/block",
+  "elements": [
```

Comment:
why not just set "particle": "#all" here so you don't have to worry about particle in any of the code or generated models above?

Reply:
It's only really relevant for the `default` model. In any case, all of the models now inherit from `block/cube_all` which does have `particle` bound to `#all`.